src/lib/routes/admin-api/client-metrics.ts

```diff
@@ -44,5 +52,23 @@ class ClientMetricsController extends Controller {
             ...data,
         });
     }
+
+    private static parseHoursBackQueryParam(
+        param: unknown,
+    ): number | undefined {
+        if (typeof param !== 'string') {
+            return;
```

Comment:
We''re generally using an explicit "return undefined" in this codebase. It's not a biggie but it would be nice to follow this pattern here too